src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://google.pl"
+      data-qa="big"
+    >
+      <label class="box">
+        <img
+           src="images/Search.svg"
+           alt="search icon"
+           class="search-icon"
+      >
+        <input
+        class="input"
+        type="search"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
```

Comment:
Fix indentation, attributes should have 2 spaces indentation from tag
```suggestion
        <input
          class="input"
          type="search"
          name="search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
      >
```